src/features/CourtCaseList/CourtCaseListEntry/CourtCaseListEntry.tsx

```diff
@@ -61,70 +52,49 @@ const CourtCaseListEntry: React.FC<Props> = ({
   const hasTriggers = triggers.length > 0
   const hasExceptions = !!errorReport
 
-  const exceptionsReasonCell = <ExceptionsReasonCell exceptionCounts={groupErrorsFromReport(errorReport)} />
-  const exceptionsLockTag = (
-    <ExceptionsLockTag
-      errorLockedByUsername={errorLockedByUsername}
-      errorLockedByFullName={errorLockedByUserFullName}
-      canUnlockCase={!!errorLockedByUsername && canUnlockCase(errorLockedByUsername)}
-      unlockPath={unlockCaseWithReasonPath("Exception", `${errorId}`)}
-      exceptionsHaveBeenRecentlyUnlocked={exceptionHasBeenRecentlyUnlocked}
-    />
-  )
-  const triggersReasonCell = <TriggersReasonCell triggers={triggers} />
-  const triggersLockTag = (
-    <TriggersLockTag
-      triggersLockedByUsername={triggerLockedByUsername}
-      triggersLockedByFullName={triggerLockedByUserFullName}
-      triggersHaveBeenRecentlyUnlocked={triggerHasBeenRecentlyUnlocked}
-      canUnlockCase={!!triggerLockedByUsername && canUnlockCase(triggerLockedByUsername)}
-      unlockPath={unlockCaseWithReasonPath("Trigger", `${errorId}`)}
-    />
-  )
-  const reasonAndLockTags: [JSX.Element, JSX.Element][] = []
+  let exceptionsReasonCell, exceptionsLockTag, triggersReasonCell, triggersLockTag
   if (hasExceptions && currentUser.hasAccessTo[Permission.Exceptions]) {
-    reasonAndLockTags.push([exceptionsReasonCell, exceptionsLockTag])
+    exceptionsReasonCell = <ExceptionsReasonCell exceptionCounts={groupErrorsFromReport(errorReport)} />
+    exceptionsLockTag = (
+      <ExceptionsLockTag
+        errorLockedByUsername={errorLockedByUsername}
+        errorLockedByFullName={errorLockedByUserFullName}
+        canUnlockCase={!!errorLockedByUsername && canUnlockCase(errorLockedByUsername)}
+        unlockPath={unlockCaseWithReasonPath("Exception", `${errorId}`)}
+        exceptionsHaveBeenRecentlyUnlocked={exceptionHasBeenRecentlyUnlocked}
+      />
+    )
   }
   if (hasTriggers && currentUser.hasAccessTo[Permission.Triggers]) {
-    reasonAndLockTags.push([triggersReasonCell, triggersLockTag])
+    triggersReasonCell = <TriggersReasonCell triggers={triggers} />
+    triggersLockTag = (
+      <TriggersLockTag
+        triggersLockedByUsername={triggerLockedByUsername}
+        triggersLockedByFullName={triggerLockedByUserFullName}
+        triggersHaveBeenRecentlyUnlocked={triggerHasBeenRecentlyUnlocked}
+        canUnlockCase={!!triggerLockedByUsername && canUnlockCase(triggerLockedByUsername)}
+        unlockPath={unlockCaseWithReasonPath("Trigger", `${errorId}`)}
+      />
+    )
   }
 
-  const classes = useCustomStyles()
-
   return (
-    <>
+    <tbody>
```

Comment:
I don't like the repeated `<tbody>` within a single `<table>` and `<thead>`.

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody

Reply:
I see now why it needs to be a `<tbody>` 

![Screenshot 2023-12-14 at 16 37 41](https://github.com/ministryofjustice/bichard7-next-ui/assets/1273965/adee5fec-bd62-4a68-a1ec-b8b80559c240)